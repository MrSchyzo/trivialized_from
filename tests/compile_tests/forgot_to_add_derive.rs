#[From(B)]
struct A;

struct B;

#[allow(dead_code)]
fn main() {}
