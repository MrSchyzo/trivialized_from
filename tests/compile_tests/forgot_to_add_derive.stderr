error: cannot find attribute `From` in this scope
 --> $DIR/forgot_to_add_derive.rs:1:3
  |
1 | #[From(B)]
  |   ^^^^
